tests/transport/compliance.py

```diff
@@ -61,14 +61,15 @@ def wait():
         yield server
         server.stop()
 
+    @pytest.mark.asyncio
```

Comment:
Does adding this mark here mean that these tests are no longer running on the sync version?

Reply:
Resolved